### PR TITLE
Marker position updates, sync marker height with active link

### PR DESCRIPTION
**Pull request by friartuck6000**

### CODE REVIEW

**Comment by billcolumbia** [comment]
I just saw this... isn't it supposed to email me!?

File: src/butr.js [hunk 1]
```diff
@@ -469,13 +469,15 @@ export const marker = options => {
     marker = document.createElement('div')
     marker.classList.add('js-butr-marker')
     if (settings.markerClass) appendClasses(marker, settings.markerClass)
-    marker.style.height = links[0].offsetHeight + 'px'
+    marker.style.height = `${links[0].offsetHeight}px`
```
Comment (billcolumbia) [inline src/butr.js]: You would 😂

File: src/butr.js [hunk 2]
```diff
@@ -486,7 +488,13 @@ export const marker = options => {
...
   const setMarkerPosition = activeLink => {
-    marker.style.transform = `translateY(${activeLink.offsetTop}px)`
+    let translatePos = activeLink.offsetTop
+    const style = window.getComputedStyle(activeLink)
+    if (style.getPropertyValue('box-sizing') === 'border-box') {
+      translatePos -= Math.round(parseFloat(style.getPropertyValue('border-top-width'), 10))
+    }
+    marker.style.transform = `translateY(${translatePos}px)`
+    marker.style.height = `${activeLink.offsetHeight}px`
   }
```
Comment (billcolumbia) [inline src/butr.js]: 👍

Comment (billcolumbia) [reply src/butr.js]: Didn't know you would need to round a border width that was _returned_ by the damn browser...

**Comment by billcolumbia** [comment]
Why didn't we ever merge this @friartuck6000

**Comment by friartuck6000** [comment]
Because #life

**Pull request closed without merging**